packages/cli-platform-apple/src/commands/runCommand/createRun.ts

```diff
@@ -51,7 +51,9 @@ const createRun =
   async (_: Array<string>, ctx: Config, args: FlagsT) => {
     // React Native docs assume platform is always ios/android
     link.setPlatform('ios');
-    const platformConfig = ctx.project[platformName] as IOSProjectConfig;
+    const platformConfig = ctx.project[platformName] as
+      | IOSProjectConfig
+      | undefined;
```

Comment:
I think we can also skip casting this type to undefined.

```suggestion
    const platformConfig = ctx.project[platformName] as IOSProjectConfig;
```

Reply:
@szymonrybczak I've just pushed a new commit you suggested